test/test_base.py

```diff
@@ -208,14 +239,43 @@ def test_linear_nobias(
                     f"CUDA capability {torch.cuda.get_device_capability()} < (9.0)"
                 )
                 pytest.skip()
+        if linear_type is LinearType.DYNAMIC:
+            # Only test one combination of scaling types, as they are a no-op
```

Comment:
this can be reusable but not worth it since it will be deleted further in the stack

Reply:
we do use it in a lot of places lol but thats fine

Reply:
I tried again just now to make it reusable and it was about the same LOC...:(